stylelint-polaris/tsconfig.json

```diff
@@ -5,7 +5,8 @@
     "outDir": "dist",
     "allowJs": true,
     "checkJs": true,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "noEmit": true
```

Comment:
I think we want a `composite: true, noEmit: false, emitDeclarationOnly:true` in here, so that the incremental compile can reference the d.ts files from a prior compilation.

Where does the tsbuildinfo file for this package get generated?

Reply:
Enabled composite 👍  Ty